### ISSUE TITLE
self-check fails due to missing dnspython library

### ISSUE THREAD

**Issue by frgomes** [spantaleev/matrix-docker-ansible-deploy#85]
Running a self-check like shown below...

```bash
ansible-playbook -i inventory/hosts setup.yml --tags=self-check
```
... fails with message:

```bash
TASK [matrix-base : Check DNS SRV record for _matrix._tcp on mathminds.io.] ********************************************************************************************************************************************************************
fatal: [matrix.mathminds.io]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'dig'. Error was a <class 'ansible.errors.AnsibleError'>, original message: The dig lookup requires the python 'dnspython' library and it is not installed"}
```

Notice that from command line I can find the DNS record:

```bash
$ dig +short _matrix._tcp.mathminds.io srv
10 0 8448 matrix.mathminds.io.
```

**Comment by frgomes** [comment]
FYI, I've used the script below which automates the steps pointed out in the documentation.
https://github.com/frgomes/bash-scripts/blob/master/sysadmin-install/install_matrix.sh

**Comment by spantaleev** [comment]
Fixed in 0ff6735.

Thanks for the report!
